package-lock.json

```diff
@@ -0,0 +1,7682 @@
+{
```

Comment:
This whole file is new? Is this right, or is this the sort of file that's generated on the local computer and should be in .gitignore?

Reply:
It is generated on the local computer (by running `npm install`), but it **is** supposed to be committed.  (In and of itself, it has nothing to do with this PR; it got generated because I was running `npm install` to bring in the newest version of circ-web.)